@uportal/content-carousel/src/lib/RssStrategy.ts

```diff
@@ -39,7 +39,10 @@ export class RssStrategy implements DataStrategy {
           const image = media ? media.content : enclosures ? enclosures[0] : null;
           return {
             id: `${index}-${new Date().getTime()}`,
-            altText: `${title} - ${description}`,
+            altText: title && description ?
+              `${title} - ${description}` :
+              title ? title :
+              description ? description : '',
```

Comment:
If the `altText` is empty, will the horizontal dark bar be drawn?  (I'm thinking it shouldn't.)

Reply:
No, that is what this is handling. There is a check in the template to hide the black bar with text if the altText is falsy, but the template that we had in place before would always at least put a " - " in, which would always cause it to display. This ternary makes sure title/description are unset if both are null, but can also display them individually without the " - " character if only one exists.

Reply:
Nice -- that's good.